### PR TITLE
Endpoint project analysis to consume in central bot

### PR DESCRIPTION
**Pull request by rodrigovirgilio**

### CODE REVIEW

File: app/api/entities/project_analysis.rb [hunk 1]
```diff
@@ -2,6 +2,22 @@ class Entities::ProjectAnalysis < Entities::BaseEntity
   expose :velocity
   expose :volatility
   expose :current_iteration_number
+  expose :date_for_iteration
```
Comment (andrerpbts) [inline app/api/entities/project_analysis.rb]: This method needs a better name. Perhaps, `next_iteration_date` describes better what the method is doing.

File: app/api/entities/team.rb [hunk 2]
```diff
@@ -3,9 +3,16 @@ class Entities::Team < Entities::BaseEntity
...
 
   with_options(format_with: :iso_timestamp) do
     expose :created_at, if: lambda { |i, o| i.created_at.present? }
     expose :archived_at, if: lambda { |i, o| i.archived_at.present? }
   end
+
+  private
+
+  def admins
+    object.enrollments.includes(:user).where(is_admin: true).map(&:user) rescue []
```
Comment (andrerpbts) [inline app/api/entities/team.rb]: Using this forces you to includes `:enrollments`in Teams query. Make sure you are adding it in necessary places.

File: app/api/entities/project.rb [hunk 3]
```diff
@@ -28,4 +30,8 @@ def iteration_service
       object.iteration_service(since: options[:since].to_i.months.ago)
     end
   end
+
+  def teams
+    object.teams
```
Comment (andrerpbts) [inline app/api/entities/project.rb]: Using this forces you to include `:teams` in Projects query. Make sure you are adding it in necessary places.

File: app/api/entities/team.rb (hunk 2 above)
Comment (rodrigovirgilio) [inline app/api/entities/team.rb]: In this case it was necessary

File: app/api/entities/project.rb (hunk 3 above)
Comment (rodrigovirgilio) [inline app/api/entities/project.rb]: In this case it was necessary

**Pull request merged**